### PR TITLE
[ AI Deploy ] - apps portfolio

### PR DESCRIPTION
**Pull request by eleapttn**
Create a new doc for AI Deploy Beta in order to propose a portfolio of AI apps and models:

- list some open-source AI apps or models that can be deployed
- add our own references (ai-training-examples repo + doc)
- duplicata EN done ✅

### CODE REVIEW

File: pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+> [!primary]
+>
+> AI Deploy is in `beta`. During the beta-testing phase, the infrastructure’s availability and data longevity are not guaranteed. Please do not use this service for applications that are in production, as this phase is not complete.
+>
+> AI Deploy is covered by **[OVHcloud Public Cloud Special Conditions](https://storage.gra.cloud.ovh.net/v1/AUTH_325716a587c64897acbef9a4a4726e38/contracts/d2a208c-Conditions_particulieres_OVH_Stack-WE-9.0.pdf)**.
+>
+
+## Objective
+
+AI Deploy allows you to deploy AI apps or models. To test or use the product, you can build on existing AI models.
```
Comment (philippart-s) [inline pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md:20]: It's just a proposition: as we have _apps_ option in our CLI and that was the old name of _AI Deploy_, I think it's better to call it _applications_ instead _apps_

```suggestion
AI Deploy allows you to deploy AI applications or models. To test or use the product, you can build on existing AI models.
```

File: pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+For example, you can rely on **open-source** models or apps.
```
Comment (philippart-s) [inline pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md:22]: cf previous remark

```suggestion
For example, you can rely on **open-source** models or applications.
```

File: pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+
+## Portfolio of AI apps and models
```
Comment (philippart-s) [inline pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md:24]: cf previous remark

```suggestion
## Portfolio of AI applications and models
```

File: pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+
+Below are examples of existing models that can be deployed with **AI Deploy**. However, you are free to deploy any other AI model or app of your choice.
```
Comment (philippart-s) [inline pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md:26]: cf previous remark

```suggestion
Below are examples of existing models that can be deployed with **AI Deploy**. However, you are free to deploy any other AI model or application of your choice.
```

File: pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+You can also refer to our [GitHub repository](https://github.com/ovh/ai-training-examples) to find examples of AI apps to deploy.
+
+> [!primary]
+>
+> You will find all the **codes** and **documentation** needed to deploy each app [here](https://docs.ovh.com/gb/en/publiccloud/ai/).
+>
+
+Here are some examples of AI apps we propose:
+
+- Deploy an app for audio classification task using Streamlit
```
Comment (philippart-s) [inline pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md:118]: cf previous remark

```suggestion
- Deploy an application for audio classification task using Streamlit
```

File: pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+- Deploy a web service for YOLOv5 using Flask
+- Deploy a Gradio app for sketch recognition
```
Comment (philippart-s) [inline pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md:120]: cf previous remark

```suggestion
- Deploy a Gradio application for sketch recognition
```

File: pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+- Deploy an app for sentiment analysis with Hugging Face models using Flask
```
Comment (philippart-s) [inline pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md:121]: cf previous remark

```suggestion
- Deploy an application for sentiment analysis with Hugging Face models using Flask
```

File: pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+- Deploy an interactive app for EDA and prediction using Streamlit
```
Comment (philippart-s) [inline pages/platform/ai/deploy_guide_05_app_portfolio/guide.en-gb.md:122]: cf previous remark

```suggestion
- Deploy an interactive application for EDA and prediction using Streamlit
```

**Review by philippart-s — APPROVED** [review]
Up to you to accept or not my suggestions 😉

**Comment by eleapttn** [comment]
@philippart-s thanks for the doc review! 

For the moment we are keeping the `app` name that is in most of the docs for simplicity. We are however interested to discuss this name for the future of the AI Deploy product 😉 .

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @eleapttn

**Pull request merged**